### ISSUE TITLE
Enhancement: Support `/Lang` (taken from HTML `lang` attribute)

### ISSUE THREAD

**Issue by ThomasLandauer** [dompdf/dompdf#1750]
It looks like the generated PDF doesn't have a natural language declaration, which is a suggested feature for better accessibility: https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/PDF16

I didn't do any thorough testing, but I'm seeing that when I have a `lang` attribute on the `<html>` tag, it has no effect in dompdf.
